virtcontainers/mount_test.go

```diff
@@ -311,7 +311,7 @@ func TestIsEphemeralStorage(t *testing.T) {
 	}
 	defer os.RemoveAll(dir)
 
-	sampleEphePath := filepath.Join(dir, k8sEmptyDir, "tmp-volume")
+	sampleEphePath := filepath.Join(dir, K8sEmptyDir, "tmp-volume")
```

Comment:
The type is k8sEmptyDir, not K8sEmptyDir, right?

Reply:
It's this const - https://github.com/kata-containers/runtime/blob/9628c2493bd2e6760b1ca75eb8185b28d079e0e1/virtcontainers/mount.go#L347

Which is also used here - https://github.com/kata-containers/runtime/blob/9628c2493bd2e6760b1ca75eb8185b28d079e0e1/virtcontainers/mount.go#L389